fenix/app/nimbus.fml.yaml

```
@@ -142,6 +142,13 @@ import:
         }
 
 features:
+  toolbar:
+    description: The searchbar/awesomebar that user uses to search.
+    variables:
+      toolbar-position-top:
+        description: If true, toolbar appears at top of the screen.
+        type: Boolean
+        default: false
```

If I remember correctly, you wanted to enable it for `nighty` and `developer` channels.

You can do that with:

```suggestion
        default: false
    defaults:
      - channel: nightly, developer
        value:
          toolbar-position-top: true
```

---

That's right! I missed that. Thank you